### DISCUSSION TITLE
clx & cly

### DISCUSSION THREAD

**Discussion by polluks2** [cc65/cc65#1486] [Ideas]
How about two new macros for generic.mac to get subtle 6280 support?

**Comment by mrdudz** [comment]
Whats the intend? I dont like the idea of using "alternative mnemonics" at all for that matter (also blt and bgt are terrible in my eyes for that matter).

**Reply by polluks2** [reply]
I know what you mean. But it would make things easier, see https://github.com/cc65/cc65/commit/4c37f12a4d5a35e113a59b9f8299801f616564da.

**Reply by mrdudz** [reply]
It'd only make it more confusing to me. I'd have too look up wth "clx" is every time i see it. Just like i have to do with bgt and blt - its confusing for anyone who already knows 6502 asm.

**Reply by oliverschmidt** [reply]
I only see the points in macros with "more added value" like ldax and stax. For them the benefit outweights the learning curve.

**Comment by oliverschmidt** [comment]
> I dont like the idea of using "alternative mnemonics" at all for that matter (also blt and bgt are terrible in my eyes for that matter).

Me too.

**Comment by groessler** [comment]
I don't like ldax and stax neither. But that's probably just me...

**Reply by mrdudz** [reply]
+1 for this :) That kind of macros do more harm than good, imho

**Comment by polluks2** [comment]
ok, ok...
